src/components/calc.js

```diff
@@ -1,52 +1,44 @@
-  <button type="button" id = "5" className="button" onClick={() => this.handleClick('5')}>5</button>
-  <button type="button" id = "6" className="button" onClick={() => this.handleClick('6')}>6</button>
-  <button type="button" id = "-" className="button orange" onClick={() => this.handleClick('-')}>-</button>
-  <button type="button" id = "1" className="button" onClick={() => this.handleClick('1')}>1</button>
-  <button type="button" id = "2" className="button" onClick={() => this.handleClick('2')}>2</button>
-  <button type="button" id = "3" className="button" onClick={() => this.handleClick('3')}>3</button>
-  <button type="button" id = "+" className="button orange" onClick={() => this.handleClick('+')}>+</button>
-  <button type="button" id = "0" className="button double-button" onClick={() => this.handleClick('0')}>0</button>
-  <button type="button" id = "." className="button" onClick={() => this.handleClick('.')}>.</button>
-  <button type="button" id = "=" className="button orange" onClick={() => this.handleClick('=')}>=</button>
+  <button type="button" id = "AC" className="button" onClick={() => handleClick('AC')}>AC</button>
+  <button type="button" id = "+/-" className="button" onClick={() => handleClick('+/-')}>+/-</button>
+  <button type="button" id = "%" className="button" onClick={() => handleClick('%')}>%</button>
+  <button type="button" id = "&divide;" className="button orange" onClick={() => handleClick('÷')}>÷</button>
+  <button type="button" id = "7" className="button" onClick={() => handleClick('7')}>7</button>
+  <button type="button" id = "8" className="button" onClick={() => handleClick('8')}>8</button>
+  <button type="button" id = "9" className="button" onClick={() => handleClick('9')}>9</button>
+  <button type="button" id = "&times;" className="button orange" onClick={() => handleClick('x')}>x</button>
+  <button type="button" id = "4" className="button" onClick={() => handleClick('4')}>4</button>
+  <button type="button" id = "5" className="button" onClick={() => handleClick('5')}>5</button>
+  <button type="button" id = "6" className="button" onClick={() => handleClick('6')}>6</button>
+  <button type="button" id = "-" className="button orange" onClick={() => handleClick('-')}>-</button>
+  <button type="button" id = "1" className="button" onClick={() => handleClick('1')}>1</button>
+  <button type="button" id = "2" className="button" onClick={() => handleClick('2')}>2</button>
+  <button type="button" id = "3" className="button" onClick={() => handleClick('3')}>3</button>
+  <button type="button" id = "+" className="button orange" onClick={() => handleClick('+')}>+</button>
+  <button type="button" id = "0" className="button double-button" onClick={() => handleClick('0')}>0</button>
+  <button type="button" id = "." className="button" onClick={() => handleClick('.')}>.</button>
+  <button type="button" id = "=" className="button orange" onClick={() => handleClick('=')}>=</button>
```

Comment:
- [ ] (Optional) It would be nice to put all button symbols in an array, then map through the array instead of repeating codes.

_**Hint**_: 
```
<div>
            {buttonSymbols.map((button) => (
              <button
                key={button}
                onClick={() => handleClick(button)}
                type="button"
              >
                {button}
              </button>
            ))}
          </div>
```